src/DSharp.Compiler/CodeModel/Members/ParameterNode.cs

```diff
@@ -33,5 +36,7 @@ internal sealed class ParameterNode : ParseNode
         public string Name => name.Name;
 
         public ParseNode Type { get; }
+
+        public bool ContainsThis { get; }
```

Comment:
IsExtensionMethodTarget

Reply:
This actually maps with Monos output ;) However I prefer your name!